win32/src/win32credmodule.cpp

```diff
@@ -82,7 +82,8 @@ BOOL PyWinObject_AsCREDENTIAL_ATTRIBUTE(PyObject *obattr, PCREDENTIAL_ATTRIBUTE
         if (PyUnicode_AsWideChar(obValue, (wchar_t *)attr->Value, nchars) == -1) {
             goto done;
         }
-    } else {
+    }
```

Comment:
For future reference, please leave the existing conventions alone where you can - I hate this indentation style :)

Reply:
I could not agree more with you regarding the indent style.
However, it wasn't me who changed it (not voluntarily at least), but the *CLang* formatting tool (it was the last place in the file which wasn't compliant). Hmm, will take a look at the spec file to see if this thing (alone) can be changed.

Reply:
welp, I gotta admit that consistency enforced by a tool does tend to trump my personal preference, so I guess I'm fine with that :)